packages/sn-filter-pane/src/components/ListboxGrid/use-handle-resize.ts

```diff
@@ -146,10 +172,11 @@ export default function useHandleResize({
   };
 
   const resourceIds = resources.map((item) => item?.id).join(',');
+  const resourceLayouts = resources.map((item) => item?.layout); // make it react to new layout objects (fetched using "pure layout")
```

Comment:
isn't this creating a new `resourceLayouts` array on each render? (making the useCallback unnecessary)

Reply:
oh, yes, you're right about that.. will adjust that

Reply:
Done, think it should be fine now